src/config.js

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
+
+const { PORT = '1337', HOST = 'localhost' } = process.env;
+
+//You need to tap $env:DEBUG="config" in terminal to turn on debug
```

Comment:
Nice to see that you use English for comments 😄 

Could you please also add `NOTE:`, `HACK:`or `TODO:`before an actual description? We follow this convention in the TestCafe code.

Reply:
Thank you)
Did it